report/src/system_evaluation/results.tex

```diff
@@ -1,2 +1,47 @@
-\section{Results}\label{sec:experiment_results}
-%describe in terms of criteria for using bluetooth. `'
\ No newline at end of file
+\section{System evaluation}\label{sec:experiment_results}
+%describe in terms of criteria for using bluetooth. `'
+The collection of measurements using the app is carried out in the same room as the map is created. 
+However, while the room was unoccupied during the creation of the map, this was not the case for the 
+second part of the experiment. 
+During this experiment, computers and phone where placed near the tables of the room, with several of them having either only wifi or Bluetooth and Wifi enabled. 
+In the common area outside the room, two people were tasked with moving around, and a laptop with Bluetooth and Wifi enabled was placed on one of the tables.
```

Comment:
```suggestion
In the common area outside the room, two people were tasked with moving around, and a laptop with Bluetooth and WiFi enabled was placed on one of the tables.
```

Reply:
This sounds unstructured. What are we trying to say here and what is the purpose?